Airports/EGSS/Ground_Network.txt

```diff
@@ -1,10 +1,153 @@
+EXIT:22:P:LEFT:25
+COORD:N051.53.26.425:E000.14.35.823
+COORD:N051.53.26.025:E000.14.35.351
+COORD:N051.53.25.730:E000.14.35.128
+COORD:N051.53.25.438:E000.14.35.033
+COORD:N051.53.25.050:E000.14.35.102
+COORD:N051.53.24.461:E000.14.35.582
+COORD:N051.53.23.921:E000.14.36.505
+COORD:N051.53.22.024:E000.14.39.801
```

Comment:
Have you tested this one? Having P as a 22 Exit seems a little strange to me, but if the airliners ignore it, that's fine.